client/app/vps/modal/shortcut/kvm/novnc/novnc.module.js

```diff
@@ -0,0 +1,11 @@
+import component from './novnc.component';
+
+import './index.less';
+
+const moduleName = 'ovhManagerVpsKvmNovnc';
+
+angular.module(moduleName, [])
```

Comment:
Shouldn't add `'oui'` as module dependency?

Because you are using some `<oui-` component.
Or is it planned to do it later on maybe?

Thanks!

Reply:
You use translations as well :)